manifest.base.yml

```diff
@@ -1,13 +1,14 @@
 ---
 buildpack: python_buildpack
-command: python manage.py migrate --fake-initial && waitress-serve --url-prefix /regulations --port=$VCAP_APP_PORT fec_eregs.wsgi:application
+command: python manage.py migrate --fake-initial && gunicorn fec_eregs.wsgi:application
```

Comment:
I believe gunicorn will pick up the host and port from the environment variables automatically, but it's always a good idea to be explicit. I don't think cloud.gov generally assigns port 8000.